documentation/modules/rn-2.6.adoc

```diff
@@ -58,6 +58,22 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of static IPs in migrations from vSphere
+
+You can now preserve static IPs of VMs that are installed with a Windows operating system and are migrated from vSphere. link:https://issues.redhat.com/browse/MTV-1148[(MTV-1148)]
+
+.Support for migrating LUKS-encrypted devices in migrations from vSphere
+
+You can now migrate devices that have been encrypted by Linux Unified Key Setup (LUKS) when you create a vSphere migration plan. link:https://issues.redhat.com/browse/MTV-831[(MTV-831)]
+
+.Specifying the primary disk when migrating from vSphere
+
+You can now specify the primary disk when you migrate VMs from vSphere which have more than one single bootable disk. This avoids {project-short} automatically attempting to convert the first bootable disk that it detects while it examines all the disks in a migration plan. This feature is needed because the first bootable disk is not necessarily the disk that the VM is expected to boot from in {virt}. link:https://issues.redhat.com/browse/MTV-1079[(MTV-1079)]
```

Comment:
```suggestion
You can now specify the primary disk when you migrate VMs from vSphere which have more than one bootable disk. This avoids {project-short} automatically attempting to convert the first bootable disk that it detects while it examines all the disks of a virtual machine. This feature is needed because the first bootable disk is not necessarily the disk that the VM is expected to boot from in {virt}. link:https://issues.redhat.com/browse/MTV-1079[(MTV-1079)]
```
(1) `one single` doesn't sound right, was it intentionally written this way?
(2) the examination is of the disks of a virtual machine (not of those in a migration plan)